src/index.html

```diff
@@ -7,8 +7,56 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <a href="home.html" class="logo">
+          <img src="images/logo.png" alt="Moyo logo">
+        </a>
+
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="Apple.html" class="nav__link is active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="Samsung.html" class="nav__link is active">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="smartphones.html" class="nav__link is active">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="laptops.html"
+                class="nav__link is active"
+                data-qa="hover"
+                >
+                Laptops&Computers</a>
+            </li>
```

Comment:
Fix it with code style
```suggestion
            <li class="nav__item">
              <a
                href="laptops.html"
                class="nav__link is active"
                data-qa="hover"
              >
                Laptops&Computers
              </a>
            </li>
```